libtiff/bitarray-0.3.5-numpy/bitarray/test_bitarray.py

```diff
@@ -16,7 +16,7 @@
     from __init__ import bitarray, bits2bytes
     repr_type = "<class '__init__.bitarray'>"
 else:
-    from bitarray import bitarray, bits2bytes
+    from libtiff.bitarray import bitarray, bits2bytes
```

Comment:
I tried this as relative, and it did not work. In tests, I suppose, it makes sense to use absolute, as tests should pretend to act as a user.